### PR TITLE
feat [component]: Add contributors page in Website

### PR DESCRIPTION
**Pull request by yashksaini-coder**
✅ Fixes: #133 

This PR basically adds new nav config, with a page where all the contributors profiles are also shown, once it is merged with the original branch. Also added rosters for `stargazers` & `forkers`. 

Plus as a test I created a stargazers mp4 or gif animation media files, using remotion. Also it is currently attached in the [README.md](https://github.com/yashksaini-coder/sneaky-package/blob/yash/fix-133/README.md)

And proper [action](https://github.com/yashksaini-coder/sneaky-package/blob/yash/fix-133/.github/workflows/add-contributors.yml) workflows to fetch & generate contributors table on the [contributors.md](https://github.com/yashksaini-coder/sneaky-package/blob/yash/fix-133/docs/CONTRIBUTORS.md)

---

@this-is-yaash @Wahid7852 Review this PR & add proper labels to it.

### CODE REVIEW

**Comment by yashksaini-coder** [comment]
For the new changes go to [here](https://yashksaini-coder.github.io/sneaky-package/CONTRIBUTORS/)

**Comment by techy4shri** [comment]
A contributor page is unnecessary since GitHub already lists contributors. Recognizing contributions is important, but duplicating it on the website is redundant and needless.
@this-is-yaash @Wahid7852 @jindalpriyanshu101 @ShashwatAgrawal20 you can close this.

File: assets/main.gif
Comment (this-is-yaash) [inline assets/main.gif:1]: this looks cool tho!
but I don't think we need this.

**Review by this-is-yaash — COMMENTED** [review]
I have reviewed this PR and I have also visited the site you mentioned.

What I think is that we don't need a contributor's page in "**documentation site**". We just need a **bare static site that holds information regarding the project** (execution, setup, guidelines etc..)

At first the idea, seemed "okay". But I personally think adding contributor's page holds no value (also not relevant) for the documentation site.

**Pull request closed without merging**